<?xml version='1.0'?>
<sdf version="1.4">
<model name="yellow_cylinder">
  <pose>0 0 0.1 0 0 0</pose>
    <link name="sample_link">
      <gravity>false</gravity>
      <inertial>
        <mass>0.5</mass>
        <inertia>
          <ixx>0.0029</ixx>       
          <ixy>0.0</ixy>      
          <ixz>0.0</ixz>         
          <iyy>0.0029</iyy>       
          <iyz>0.0</iyz>     
          <izz>0.0025</izz>      
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <cylinder>
            <radius>0.10</radius>
            <length>0.2</length>
          </cylinder>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <cylinder>
            <radius>0.10</radius>
            <length>0.2</length>
          </cylinder>
        </geometry>
        <material>
        	<script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Yellow</name>
            </script>
        </material>
      </visual>
    </link>
  </model>
</sdf>

